app/views/user_mailer/abuse_report.html.erb

```diff
@@ -21,7 +21,7 @@
 
   <p>
     <%= style_work_metadata_label(t(".copy.summary")) %>
-    <%= raw @summary %>
+    <%= to_plain_text(raw @summary) %>
```

Comment:
If we're planning to fix this differently in [AO3-6519](https://otwarchive.atlassian.net/browse/AO3-6519), can you add a TODO here?